.travis.yml

```diff
@@ -39,11 +36,10 @@ install:
   - |
     if [[ $TRAVIS_OS_NAME == "osx" ]]
     then
-      brew update || brew update # try again if it fails
-      brew upgrade
-      brew reinstall openssl
-      brew reinstall pyenv
+      pyenv -v
```

Comment:
We deal with brew here because pyenv is already installed by brew, so running pyenv's provided installer is duplicating things.  Also, curl | bash is awful, and pyenv's installer is meh anyway.